kubens

```diff
@@ -54,14 +54,15 @@ get_namespaces() {
 }
 
 namespace_file() {
-  local ctx="${1}"
+  local ctx="${1//\//-}"
   echo "${KUBENS_DIR}/${ctx}"
 }
 
 read_namespace() {
   local f
   f="$(namespace_file "${1}")"
   [[ -f "${f}" ]] && cat "${f}"
+  return 0
```

Comment:
I'm trying to understand why this exists. Can you explain?

Reply:
As you said in  6610d70, the https://github.com/koalaman/shellcheck/wiki/SC2155 prevents masking the return value. In case $f does not exists the return value is 1, and since you're using `set -eou` this aborts the script.

A non-existent file in this case is just fine, so I again mask the return value

Reply:
I see it now when I executed it myself with `DEBUG=1 ./kubens`. Thanks so much for noticing this.

I guess this works, because in both places `read_namespace` is called, empty output will do the job.